src/structs/selections.py

```diff
@@ -18,7 +18,7 @@ class ImageCropSelection:
     img_size: tuple[int, int]
     crop_position: Tuple[int, int, int, int]
 
-    def to_dict(self) -> Dict[str, str]:
+    def to_dict(self) -> Dict[Any, Any]:
```

Comment:
Consider using the `dict` typehint without subscript types (`Dict[Any, Any]` is roughly equivalent to an unconstrained `dict`).
```suggestion
    def to_dict(self) -> dict:
```